src/types.ts

```diff
@@ -18,4 +19,14 @@ export interface PersistentStorage<T> {
 export interface Scraper {
     start(): Promise<any>;
     end(): Promise<any>;
-}
\ No newline at end of file
+}
+
+export interface ConfigLoader {
```

Comment:
Probably use `class` is better. https://stackoverflow.com/questions/51716808/when-use-a-interface-or-class-in-typescript

Reply:
We can use class for IoC binding, but if you want to have two different ConfigLoader implementation for DI, then interface is better.
class is supported: https://github.com/inversify/InversifyJS/blob/master/wiki/classes_as_id.md
but we need to sure there is no need for a different implementation for this type.